docs/decisions/0004-release-captain.md

```diff
@@ -16,34 +16,101 @@ taking appropriate measures to include all necessary fixes and features as well
 as delivering sane and functional container images. The release captain is also
 responsible for describing the changes of the new release in the release notes.
 
+### Prepare a Release
+
+For the testing phase that is done right before the official release, the
+following actions need to be done.
+
+- Create a branch for a given release in the `ceph`, `s3gw`, `s3gw-ui`,
+  `s3gw-tools` and `s3gw-charts` repositories. This can be done easily via
+  the GitHub branches page. E.g. visit the [s3gw-ui branches][1] page, click
+  the `New branch` button and enter `v0.8.0`.
+  All bugfixes will be done in the `main` branches of the sub-projects and
+  then backported to the `vx.y.z` release branch afterwards. The name of the
+  branch should reflect the current milestone (i.e. `v0.8.0`).
+  In the `ceph` repository we need to prepend `s3gw-` (i.e. `s3gw-v0.8.0`).
+- Update the chart version to `x.y.z` in `s3gw-charts/charts/s3gw/Chart.yaml`.
+  Create a PR including these changes for the `vx.y.z` release branch.
+
 ### Create a Release
 
-- Create a release branch for a given release for the `ceph` & `s3gw-ui`
-  repositories. All bugfixes will be backported to the relevant repo. The name
-  of the branch should reflect the current milestone (ie `0.8.0`). In the Ceph
-  repo we need to prepend `s3gw-`(ie `s3gw-0.8.0`)
-- Aggregate changelog and create s3gw/docs/release-notes/s3gw-vx.x.x.md Use
-  previous release notes for guidance.
-- Change s3gw/docs/release-notes/latest to point to the new release notes (this
-  will be used by automation)
-- Update all subrepos changelog from `Unreleased` to the current release date.
-- Bump all subrepos repos to the desired git ref
-- Push these changes
-- Create a version tag in the s3gw repo (this triggers the release pipeline,
-  which creates the container and a draft release)
+After the testing phase the following actions need to be done:
+
+- Update the changelog from `Unreleased` to the current release date for each
+  sub-project `s3gw-ui`, `s3gw-tools` and `s3gw-charts`. Create a PR for the
```

Comment:
needs to be done for `ceph.git` as well.

Reply:
Done